devel/retdec/Portfile

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+categories          devel
+license             MIT
+platforms           darwin
+
+description         RetDec is a retargetable machine-code decompiler based on LLVM.
+
+long_description    RetDec is a retargetable machine-code decompiler based on LLVM.\
+                    The decompiler is not limited to any particular target architecture, \
+                    operating system, or executable file format.
+
+maintainers         nomaintainer
+
+github.setup        avast retdec 4.0 v
+
+revision            0
+
+checksums           rmd160  305e901b37441337d5c0dada1ccc5e737831ef1e \
+                    sha256  b0e3172d9d232899b71dac41446592c03abe089b8e9c7c1a9c8e8bfe92cf8b49 \
+                    size    8732003
+
+depends_build       port:autoconf \
+                    port:automake \
+                    port:cmake \
+                    port:doxygen \
+                    port:git \
```

Comment:
is `git` really a build dependency?

Reply:
Yes, it is. `cmake` uses `git` to download some dependencies from github.

Reply:
that might warrant a closer look: fetching things should ideally only happen in the "fetch" phase. I guess here it is happening in the "build" phase which is not recommended - can these dependencies be added to MacPorts and used instead of downloading them during the build? I'll check the logs later once I am behing a computer

Reply:
@reneeotten it is not just building a dependenices, it building specified forked version of dependicies.

For example it requires to use specified version of LLVM (quite old) with some patches to expose some API.

And it requires sources for this dependencies :)

Thus, fetching and patching dependecies is deeply integrated into cmake's build and I have no idea how to simple run it on fetch stage. Ok, I have some ideas but it requires nearly to rework build system that seems like out of scope for portfile.